README.md

```diff
@@ -55,5 +55,32 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 ##Configuring Trello API
 First you need to create an account https://trello.com/signup
 Then you need to generate API key and token (explained here https://trello.com/app-key). The link will display your key and you need to generate manually a token.
-Add them to .env file as variables TOKEN = {Your token} and KEY = {Your key}.
-In app.py you need to import os and you can call them as 'key': os.environ['KEY'] and 'token' : os.environ['TOKEN']
\ No newline at end of file
+Put the variables to .env template as variables TOKEN = {Your token} and KEY = {Your key}.
+In app.py you need to import os and you can call them as 'key': os.environ['KEY'] and 'token' : os.environ['TOKEN']
+In order to access the id of the lists on the board: first - you need to put them to .env template as:
+toDoId = {listId}
+doingId = { listId}
+doneId = {listId}
+Those are the ones that should be assigned in the code rather than variable names.
+In app. py they get called the following way:
+toDoId = os.environ['TOID']
+doingId = os.environ['DOINGID']
+doneId = os.environ['DONE']
+
+
+##Pytest
```

Comment:
Your md formatting is off.

You can see it with styling in VSCode via a button in the top right or you can see it here:
https://github.com/I-Georg/DevOps-Course-Starter/blob/342ef45111641bfb7242049f3ae67301df3520e4/README.md